eureka_ml_insights/data_utils/transform.py

```diff
@@ -302,3 +303,40 @@ def transform(self, df: pd.DataFrame, encoding="cl100k_base") -> pd.DataFrame:
             token_count_column = f"{column}_token_count"
             df[token_count_column] = token_count
         return df
+
+@dataclass
+class MajorityVoteTransform:
+    """Applies the majority vote transformation to the 'model_output' column per 'ID'."""
```

Comment:
please make the column names arguments so the user can set them. the default values can be "model_output" and "data_point_id" which are standard Eureka columns.

Reply:
added